lib/views/index.erb

```diff
@@ -1,30 +1,56 @@
 <% if @my_cart.count != 0 %>
-  <div id="shopping_cart">
-    Shopping cart
+<div class="cell">
+  <div class="card shadow radius padding-1">
+    <h1><i class="fa fa-shopping-basket" aria-hidden="true"></i> Shopping Cart:</h1>
 
-    <ul>
-      <% @my_cart.each do |item| %>
-        <li>
-          <%=item.dish_name%>
-          <%=item.quantity%>
-        </li>
-      <%end%>
-    </ul>
-    <a href="./clear_cart">Empty the cart!</a>
+      <table class="hover">
+        <tr>
+          <th class="left">Dish</th>
+          <th class="center">Quantity</th>
+          <th class="right">Price</th>
+          <th class="right">Sum</th>
+        </tr>
+        <% @my_cart.each do |item| %>
+        <tr>
+          <td class="left"><%=item.dish_name%></td>
+          <td class="center"><%=item.quantity%></td>
+          <td class="right">150 kr</td>
+          <td class="right">300 kr</td>
+        </tr>
+        <%end%>
+        <tr>
+          <td class="left"></td>
+          <td class="center"></td>
+          <td class="right"><strong>Total sum:</strong></td>
+          <td class="right"><strong>450</strong></td>
+        </tr>
+      </table>
+      <div class="inline">
+        <a href="/clear_cart" class="button lined float-left"><i class="fa fa-trash"></i>&nbsp;Empty Shopping Cart&nbsp;</a>
+        <a href="#" class="button lined float-right"><i class="fa fa-shopping-cart"></i>&nbsp;Check Out Items&nbsp;</a>
+      </div>
+    </div>
```

Comment:
Please make sure your code is properly formatted. It makes it more readable and easier to review.